src/test/java/teammates/test/cases/ui/browsertests/AllAccessControlUiTests.java

```diff
@@ -214,7 +214,8 @@ private void verifyRedirectToWelcomeStrangerPage(AppUrl url, String unregUsernam
         currentPage.navigateTo(url);
         // A simple regex check is enough because we do full HTML tests
         // elsewhere
-        AssertHelper.assertContainsRegex("{*}" + unregUsername + "{*}Welcome stranger{*}",
+        AssertHelper.assertContainsRegex(
+                "{*}" + unregUsername + "{*}Ooops! Your Google account is not known to TEAMMATES{*}",
```

Comment:
Actually, even it was done this way before, the leading and trailing `{*}` should not be necessary, right?